src/main/java/com/sap/sgs/phosphor/fosstars/model/score/oss/ProjectActivityScore.java

```diff
@@ -26,45 +29,76 @@
 public class ProjectActivityScore extends FeatureBasedScore {
 
   /**
-   * y(n) = 7.1 / (1 + e ^ (0.04 * (50 - n))),
+   * The maximum sub-score based on a number of commits in the last three month.
+   */
+  private static final double MAX_COMMITS_SUB_SCORE = 5.1;
+
+  /**
+   * A logistic function which is used to convert a number of commits to a sub-score:
+   * <pre>y(n) = MAX_COMMITS_SUB_SCORE / (1 + e ^ (0.02 * (200 - n)))</pre>,
    * where n is a number of commits in the last three months.
-   * y(n) belongs to the interval (0.8, 7.1) when n > 0
-   * Number of commits can contribute up to 7 to the overall project activity score.
+   * The function belongs to the interval [0.0, MAX_COMMITS_SUB_SCORE) when n >= 0.
    */
-  static final Logistic LOGISTIC_FOR_NUMBER_OF_COMMITS
-      = new Logistic(7.1, 50, 0.04, 1, 0, 1);
+  private static final Logistic LOGISTIC_FOR_NUMBER_OF_COMMITS
+      = logistic(MAX_COMMITS_SUB_SCORE, 0.02, 200);
 
   /**
-   * y(m) = 3.1 / (1 + e ^ (0.2 * (1 - m))),
-   * where m is a number of contributors in the last three months.
-   * y(m) belongs to the interval (1.3, 3.1) when m > 0
-   * Number of contributors can contribute up to 3 to the overall project activity score.
+   * The maximum sub-score based on a number of contributors in the last three months.
    */
-  static final Logistic LOGISTIC_FOR_NUMBER_OF_CONTRIBUTORS
-      = new Logistic(3.1, 1, 0.2, 1, 0, 1);
+  private static final double MAX_CONTRIBUTORS_SUB_SCORE = 5.1;
 
+  /**
+   * A shift on the Y-axis for the logistic function for the number of contributors.
+   */
+  private static final double MAX_CONTRIBUTORS_SUB_SCORE_SHIFT = -1;
+
+  /**
+   * A logistic function which is used to convert a number of contributors to a sub-score:
+   * <pre>y(m) = A / (1 + e ^ (0.4 * (4 - m)))</pre>,
+   * where m is a number of contributors in the last three months,
+   * and A = MAX_CONTRIBUTORS_SUB_SCORE - MAX_CONTRIBUTORS_SUB_SCORE_SHIFT.
+   * The function belongs to the interval [0.0, MAX_CONTRIBUTORS_SUB_SCORE) when m >= 0.
+   */
+  private static final Logistic LOGISTIC_FOR_NUMBER_OF_CONTRIBUTORS
+      = logistic(MAX_CONTRIBUTORS_SUB_SCORE - MAX_CONTRIBUTORS_SUB_SCORE_SHIFT, 0.4, 4);
```

Comment:
Any specific reason these number 0.4 and 4 were used?

Reply:
It's similar to what I explained above.